_src/Anti-Debug/techniques/assembly.md

```diff
@@ -404,6 +405,20 @@ bool IsDebugged()
 
 <hr class="space">
 
+
+<br />
+<h3><a class="a-dummy" name="popf_and_cpuid">9. POPF and CPUID</a></h3>
```

Comment:
Please add the code example and make a reference to the technique number 7 in this chapter - it describes the trap flag & POPF combo